scripts/remote_replace_cloudbeat.sh

```diff
@@ -0,0 +1,35 @@
+# This script will stop any Cloudbeat process within an agent and will replace the Cloudbeat artifacts with your local built artifacts
+
+# Prerequisites:
+# a. Create the Cloudbeat artifacts using the mage command (for example - "DEV=true PLATFORMS=linux/amd64 SNAPSHOT=true mage -v package")
+# b. The running elastic-agents were built with "DEV=TRUE" flag (https://github.com/elastic/security-team/blob/main/docs/cloud-security-posture-team/Onboarding/cloudbeat-ec2.md#:~:text=Build%20the%20agent%20(We%20are%20building%20with%20DEV%3Dtrue%2C%20so%20we%20can%20load%20cloudbeat.tar.gz%20without%20signature%20verification))
+
```

Comment:
is there a way to overcome this Prerequisite? Does this basically mean we will only be able to use this script if we compiled the agent beforehand?

Reply:
If we will use an existing agent that we weren't compiled with DEV=TRUE, so the agent will throw an error that it can't validate the cloudbeat certificate.

Reply:
yeah I know I'm just trying to think of a way we can overcome this. If we could find a way to that this script would be amazing, let's talk about it next week